src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1194,6 +1203,44 @@ if b1 : 1 < b then
   le_trans (le_power_self _ b1) (power_log_le b (ordinal.pos_iff_ne_zero.2 x0))
 else by simp only [log_not_one_lt b1, ordinal.zero_le]
 
+@[simp] theorem log_one (b : ordinal) : log b 1 = 0 :=
+if hb : 1 < b then by rwa [←lt_one_iff_zero, log_lt hb zero_lt_one, power_one]
+else log_not_one_lt hb 1
+
+lemma power_mul_add_pos {b v : ordinal} (hb : 0 < b) (u) (hv : 0 < v) (w) :
```

Comment:
I just learned that lemmas about ordinal `^` are named `power`. In the rest of the library they're `pow`. No need to change in this PR but this is something that could be fixed (for all ordinal pow lemmas) in a follow up.

Reply:
There's generally some small annoying mismatches between names in this part of the library and elsewhere, of which this is the most notable. It's certainly something worth looking into, but I think it deserves its own PR.

Reply:
I actually tried "fixing" this, only to realize that it was a really bad idea. Monoids have an automatically defined `pow`, which does not match with what we want, so changing `power` to `pow` leads to many name clashes. I think it's better to leave it as is.